README.md

```diff
@@ -144,6 +145,65 @@ enum TaskStatus: int
 TaskStatus::options(); // ['INCOMPLETE' => 0, 'COMPLETED' => 1, 'CANCELED' => 2]
 ```
 
+### From
+
+This helper adds `from()` and `tryFrom()` to pure enums, and adds `fromName()` and `tryFromName()` to all enums.
+
+#### Important Notes:
+* `BackedEnum` instances already implement their own `from()` and `tryFrom()` methods, which will not be overridden by this trait. Attempting to override those methods in a `BackedEnum` causes a fatal error.
+* Pure enums do not actually have an internal index, so rearranging the order of cases will not break other code but it will create inconsistent behaviour with the `from()` and `tryFrom()` methods
```

Comment:
This means that pure enums should generally use `fromName()` rather than `from()`, right? Since the indexes are fragile due to not being explicit

Reply:
Exactly yes. Honestly I could probably drop `from()` and `tryFrom()`, but I like the "completeness" of the API.

Reply:
I agree. As I just mentioned in the other PR's review, there may be some valid use cases for this, so it makes sense to include it in a way that's consistent with backed enums. And also warn the users that it's generally not recommended.